infrastructure/tf-db-flexible-secrets.tf

```diff
@@ -1,6 +1,5 @@
 # Postgres 15 flexible server store secrets in key vault
 locals {
-  flexible_secret_prefix = "${var.component}-POSTGRES-FLEXIBLE"
```

Comment:
We could think about renaming `standard_secret_prefix` to `flexible_secret_prefix` so everything matches. Now that the old dbs have been removed, the word standard as I understand it might not be relevant anymore.

Reply:
Yeah can look at doing more of the infra refactoring as part of bulk print re-development